packages/client/src/contents/types/contents.types.ts

```diff
@@ -17,7 +18,8 @@ export type QuerySearchContents = {
   contentTypeCode: string;
   // List of codes that representing the content code (about-us|today-news|header|productId...).
   codes?: string | string[];
-  // The targets and respective values that a content type is configured (contentzone:ROW | country:GB | language:en-GB | benefits:test).
+  // The targets and respective values that a content type is configured (contentzone:ROW | country:GB | language:en-GB | benefits:test | segments: ['private-sale']).
```

Comment:
I think this comment is stale. Can you update it to something like this?

```suggestion
  // The targets and respective values that a content type is configured (contentzone:ROW | country:GB | language:en-GB | benefits:test | segments: 'private-sale, other-segment').
```

Reply:
Thanks!